samples-generator/lib/injectTranslationNodeToGltf.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
I think this solution is a bit overarchitected since the RTC node could just be manually inserted into the node hierarchy in `createGltf`. A utility like this is probably great for gltf-pipeline but I don't think this repo necessarily benefits from having it, unless we expect there to be multiple different use cases for it in here.

Reply:
Agreed. I removed `injectTranslationNodeToGltf.js` and it's associated unittest.